src/main/scripts/run_e2e_test.py

```diff
@@ -102,9 +102,46 @@ def run_shell_command(cmd):
 assert res.ok or print(res.text)
 
 if module_to_build:
+    install_driver = "false"
+    if module_to_build == 'postgresql-plugin':
+        jar_file_name = "postgresql-9.4.1211.jar"
+        driver_prop = '[{"name":"postgresql","type":"jdbc","className":"org.postgresql.Driver","description":"Plugin for the PostgreSQL JDBC driver"}]'
+        artifact_name = "postgresql"
+        artifact_version = "9.4.1211"
+        install_driver = "true"
+    elif module_to_build == 'mssql-plugin':
+        jar_file_name = "sqljdbc42.jar"
+        driver_prop = '[{"name":"sqlserver42","type":"jdbc","className":"com.microsoft.sqlserver.jdbc.SQLServerDriver","description":"Plugin for Microsoft JDBC driver 6.0 for SQL Server"}]'
+        artifact_name = "sqljdbc42"
+        artifact_version = "42"
+        install_driver = "true"
+    elif module_to_build == 'mysql-plugin':
+        jar_file_name = "mysql-connector-java-8.0.25.jar"
+        driver_prop = '[{"name":"mysql","type":"jdbc","className":"com.mysql.jdbc.Driver","description":"Plugin for the MySQL JDBC driver"}]'
+        artifact_name = "mysql-connector-java"
+        artifact_version = "8.0.25"
+        install_driver = "true"
+    elif module_to_build == 'oracle-plugin':
+        jar_file_name = "ojdbc7.jar"
+        driver_prop = '[{"name":"oracle","type":"jdbc","className":"oracle.jdbc.driver.OracleDriver","description":"Plugin for the Oracle 12c JDBC driver"}]'
+        artifact_name = "ojdbc7"
+        artifact_version = "7"
+        install_driver = "true"
```

Comment:
Instead of using dumping everything in the same script, can we have a separate `jdbc-drivers.yml` file which contains all the props like -

```
modules:
   -  module_to_build: `postgress-plugin`
          -  jar: postgresql-9.4.1211.jar
               driver_prop: .....
 ```

Reply:
Done.